react/src/components/atoms/buttons/ButtonWithIcon/style.scss

```diff
@@ -45,24 +89,149 @@
     vertical-align: middle;
     fill: $c-gray-dark;
     height: 20px;
-    transition: fill .4s;
+    transition: transform .5s, fill .4s;
   }
 
-  &.ma__icon-green > svg {
-    fill: $c-primary-alt;
+  @each $class, $c in $colors {
```

Comment:
i'm wondering if setting the icon svg color separately, as I feel like the icon should be aligned with the color of the button text and set by the theme of the button in the mixin as a whole (including hover state). I can't think of a case it's should be different on the top of my head.

Reply:
removed option to style icon with icon color prop